helper/Datatypes.php

```diff
@@ -15,12 +15,15 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ * Copyright (c) 2014-2022 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
  *
  */
 
 namespace oat\taoOutcomeUi\helper;
 
+use oat\oatbox\filesystem\utils\FlyWrapperTrait;
+use oat\taoDeliveryRdf\view\form\WizardForm;
```

Comment:
Do we really need this dependency? I could not find any usage in this file.

Reply:
we don't missclick this import and forget to remove :(